Sources/PointFree/Blog/BlogPosts/BlogPost0026_html020.swift

```diff
@@ -0,0 +1,99 @@
+import Foundation
+
+let post0026_html020 = BlogPost(
+  author: .pointfree,
+  blurb: """
+Announcing swift-html 0.2.0: support for CocoaPods, Carthage, SnapshotTesting, and more!
+""",
+  contentBlocks: [
+    .init(
+      content: """
+Today we're releasing our [first minor update](https://github.com/pointfreeco/swift-html/releases/tag/0.2.0) to [swift-html](\(gitHubUrl(to: .repo(.html)))).
+
+It contain a number of new features and fixes, including patches from the community!
+
+## What's new?
+
+### iOS Support
+
+While we imagined swift-html to be most useful on the server, [a pull request](https://github.com/pointfreeco/swift-html/pull/27) let us know that there was interest in using our library on iOS, as well! With the help of the community we now  support being embedded in iOS, tvOS, and watchOS
+
+### Carthage and CocoaPods Support
+
+After adding support for iOS we figured it'd be best to add support for its popular dependency management tools!
```

Comment:
this section is a lil meek. anyway to beef it up or combine it with the previous section under a `### Supporting to platforms` header or something?